content/en/providers/check-in/idp/_index.md

```diff
@@ -28,7 +28,7 @@ The integration follows a two-step process:
    affecting the production environment of EGI. Note that the development
    instance is not connected to the production service and no information is
    shared between the two systems.
-1. Register your Identity Provider with the production instance of EGI Check-in
+2. Register your Identity Provider with the production instance of EGI Check-in
```

Comment:
Usually it's better to rely on auto-numbering, is there any reason for introducing this manual numbering here?